include/swift/Driver/Compilation.h

```diff
@@ -175,6 +175,9 @@ class Compilation {
   /// Overrides parallelism level as count of batches, if in batch-mode.
   const Optional<unsigned> BatchCount;
 
+  /// Overrides maximum batch size, if in batch-mode.
```

Comment:
Worth mentioning that `BatchCount` wins if both are set?

Reply:
fixed